lib/node_modules/@stdlib/stats/base/dists/triangular/ctor/benchmark/benchmark.js

```diff
@@ -152,17 +167,21 @@ bench( pkg+'::set:b', function benchmark( bm ) {
 	b = 40.0;
 	c = 30.0;
 	dist = new Triangular( a, b, c );
+	len = 100;
+	y = new Float64Array( len );
+	for ( i = 0; i < len; i++ ) {
+		y[ i ] = uniform( c + EPS, c + 100.0);
```

Comment:
```suggestion
		y[ i ] = uniform( c + EPS, c + 100.0 );
```

There should be empty spaces at the beginning and end of brackets.